javascript/utils/pendingemailcredential_test.js

```diff
@@ -54,7 +54,7 @@ function setUp() {
   credentialObject = {
     'providerId': 'google.com',
     'signInMethod': 'google.com',
-    'oauthAcessToken': 'ACCESS_TOKEN'
```

Comment:
it's possible this was an intentional typo... for intentional typos, I encourage using trailing `_`s to make it clear that the item isn't the real thing...